### ISSUE TITLE
Feature Request: Recognize and properly display sub-directories

### ISSUE THREAD

**Issue by agaskins** [TapiocaFox/Daijishou#622]
Please at support to recognize folders with each system path. For example lets say I add my SNES system but have folders for Hacks, Translations, Japanese, etc.. Currently, Daijisho doesn't handle large and organized rom collections well at all.

Sure, we can manually add new systems for each of these folders, but that a huge pain to set up for a large library where most systems have these sorts of sub-folders. Just having it show the folder structure of the directory it's already using would be perfect, imo.

I feel like this is one of just a couple of features keeping the hardcore rom addicts among us from being 100% happy (don't get me wrong, I still think Daijisho is the best of what's available for Android). If it had this and dynamic collections I'd should from the mountains for people to use Daijisho! haha.

**Comment by 01siRoj** [comment]
Yep subfolders is the missing feature for this frontend. I'm not familiar whith android app but this feature is enable for another front end (ES-DE) so i think its not impossible to add it. But personaly i don't know how.

**Comment by ncflagg** [comment]
I want this also. Creating another platform, for me, results in too many platforms to scroll through. And, dumping no-name hacks in with retail releases makes the list too long and messy. This also needs to support nested folders, like if I want to include a few pages of just Mario hacks and I don't want them in the main subfolder.

**Comment by agaskins** [comment]
Thanks for the reply! Sorry I didn't get back sooner. I'm just curious if  there was ever any progress on this? I am considering getting the new Retroid 5, but I really didn't use my 4pro much because I never could find a solution in Android rom management that was ideal. Your app is sooooo close and great, but just having my hundreds of rom hacks and translations grouped in with regular roms and named generically makes it nearly impossible to find specific roms to play.

**Comment by 01siRoj** [comment]
Still nothing about this feature ?
